frontend/src/scenes/session-recordings/filters/RecordingsUniversalFiltersEmbed.tsx

```diff
@@ -581,6 +581,17 @@ const RecordingsUniversalFilterGroup = (): JSX.Element => {
                         onChange={(value) => replaceGroupValue(index, value)}
                         initiallyOpen={allowInitiallyOpen}
                         metadataSource={{ kind: NodeKind.RecordingsQuery }}
+                        operatorAllowList={
+                            isCommentTextFilter(filterOrGroup)
+                                ? [
+                                      PropertyOperator.IsSet,
+                                      PropertyOperator.Exact,
+                                      PropertyOperator.IsNot,
+                                      PropertyOperator.IContains,
+                                      PropertyOperator.NotIContains,
+                                  ]
```

Comment:
only a subset of filters make sense here

(arguably you could add `is_not_set` but that would be quite a big code change and i think not a very useful filter to support)